profile/README.md

```diff
@@ -0,0 +1,22 @@
+![Containers](https://github.com/containers/.github/blob/main/logo/containers-logo.png)
```

Comment:
We probably ought to store the logo in https://github.com/containers/common/blob/main/logos/ and for that matter, we/I should probably move the others there too.